TRX/Tron-overview.md

```diff
@@ -263,8 +267,13 @@ TRON will always support swapping ERC20 TRX to TRON Mainnet TRX.
 - For Exhanges: Please contact TRON to swap your ERC20 TRX to Mainnet TRX
 
 # 13. Super Representatives and Voting
+The Super Representatives(SR) take important roles to build and operate on TRON network such as block generation and transaction packing. They receive some TRX as rewards. Currently, for the 27 SRs, one block is generated every 3 seconds, with each block awarding 32 TRX to the 27 SRs in sequence; for the 127 SR candidates (including the 27 SRs), there would be 16 additional TRX every 3 seconds to be distributed among the 127 SR candidates in proportion to the votes they have received.
+
+Every TRX holders can vote for SR but it is required to have Tron Power which can gain from freezing TRX. You can vote SR candidates and see info about SR candidates on `votes` menu on https://tronscan.org
```

Comment:
All holders of TRX can vote for SRs as long as they have Tron Power available. Tron Power can be gained by freezing TRX. To vote for an SR candidate you can use your favourite wallet or [TronScan, the official Web wallet](https://tronscan.org/#/votes).

Reply:
how was "All TRX holders" instead of "Alll holders of TRX"?

> To vote for an SR candidate you can use your favourite wallet 

we can take a vote multiple candidates. so, how was this?

> To vote for SR candidates you can use your favorite wallet or ...

Reply:
"an SR candidate"  vs "SR candidates"

the former one looks like users can vote only one SR. But you knows, users can vote several times and multi-SRs. That's the point. but without detailed description, it looks similar both sentences. 😄

Reply:
Oh yep my bad! "SR candidates" is perfect